main/src/sphexa/sphexa.cpp

```diff
@@ -171,7 +163,7 @@ int main(int argc, char** argv)
         if (d.iteration % 50 == 0) { viz::execute(d, domain.startIndex(), domain.endIndex()); }
     }
 
-    totalTimer.step("Total execution time of " + std::to_string(d.iteration - startIteration + 1) + " iterations of " +
+    totalTimer.step("Total execution time of " + std::to_string(d.iteration - startIteration) + " iterations of " +
```

Comment:
Shouldn't the +1 be here? When iteration = startIteration that would give zero instead of 1.